package.json

```diff
@@ -39,6 +39,7 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-glob-finder": "0.0.6",
```

Comment:
Strange... I installed that when I was trying to debug using this with the local copies of the plugins, but I didn't use the save flag so I'm not sure how this got added.